baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -142,8 +156,13 @@ def __init__(
         :param queue_name: Name for your queue.
         :param routing_keys: List of routing keys that you will create :py:class:`~kombu.Queue` s
             to consume from.
-        :param handler_fn: A `baseplate.frameworks.queue_consumer.komub.Handler`
-            function that will process an individual message from a queue.
+        :param handler_fn: A function that will process an individual message from a queue.
+            If `error_handler_fn` is passed in, this function is responsible for calling
+            `message.ack` or `message.requeue` when a message is successfully processed.
```

Comment:
I don't think this is accurate, (or what we want). The only difference should be when an error is raised, `error_handler_fn` should finalize the message (`ack`, `requeue`, etc), not the case where there is no error.